addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -160,15 +160,14 @@
                         <t t-set="timesheets_amount" t-value="round(sum(timesheets.mapped('unit_amount')), 2) or 0.0"></t>
```

Comment:
While we're at it, shouldn't we use `task.effective_hours` rather than setting `timesheets_amount`? :thinking: